homeassistant/components/generic_hygrostat/strings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Generic hygrostat",
+  "config": {
+    "step": {
+      "user": {
+        "title": "Add generic hygrostat",
+        "description": "Create a entity that control the humidity via a switch and sensor.",
+        "data": {
+          "device_class": "Type of device",
```

Comment:
In the template and scrape config flow, we also use the name device class iirc. Should we stay consistent?

Reply:
The annoying thing is that our select selectors only allow string values. So if we do that we would need to adjust the  config structures. I very much dont want different structures for yaml and flow.

If our selectors allowed bool values, it would have been easily solvable, but they dont just yet.

Reply:
I think you're missing my point, I mean, now the name of the field is "type of device" (which could be "device type" imo), but should we stay consistent with HA naming and just name it "Device class"?

Reply:
Ah sorry. I got mixed up with climate that uses a boolean switch. Sure lets change that.

Reply:
```suggestion
          "device_class": "Device class",
```